### PR TITLE
Initial release v2.1.5

### PR DESCRIPTION
**Pull request by tescande**
Sources imported from [1]. Spec file created from the one provided in the source archive.

[1] https://sourceforge.net/projects/traceroute/files/

### CODE REVIEW

File: SPECS/traceroute.spec [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+%files
+%defattr(-,root,root,-)
+%doc COPYING README TODO CREDITS
+/bin/*
+%{_mandir}/*/*
+
+%changelog
+* Thu Apr 25 2024 Thierry Escande <thierry.escande@vates.tech> - 2.1.5-1
+- Initial release
```
Comment (stormi) [inline SPECS/traceroute.spec]: If this is forked from an upstream SRPM, we should retain their changelog.

Comment (stormi) [reply SPECS/traceroute.spec]: We should also import it, unmodified, in a branch named `upstream`

Comment (stormi) [reply SPECS/traceroute.spec]: I re-read your commit message, which answers this comments before I wrote them :)

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
I suggest we test it a bit before merging.

File: SPECS/traceroute.spec [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+%build
+make %{?_smp_mflags} CFLAGS="$RPM_OPT_FLAGS" LDFLAGS=""
+
+%install
+install -d $RPM_BUILD_ROOT/bin
+install -m755 traceroute/traceroute $RPM_BUILD_ROOT/bin
+pushd $RPM_BUILD_ROOT/bin
+ln -s traceroute traceroute6
+popd
```
Comment (ydirson) [inline SPECS/traceroute.spec]: we could improve to create a relative symlink without even changing dir with
```sh
ln -sr $RPM_BUILD_ROOT/bin/traceroute $RPM_BUILD_ROOT/bin/traceroute6
```

**Review by ydirson — APPROVED** [review]
small possible improvement over upstream rules, otherwise lgtm

File: SPECS/traceroute.spec (hunk 2 above)
Comment (tescande) [reply SPECS/traceroute.spec]: Done

**Comment by stormi** [comment]
Koji build to v8.3-incoming: http://koji.xcp-ng.org/taskinfo?taskID=52173

**Comment by stormi** [comment]
New build attempt after adding `gcc` to the `BuildRequires`: http://koji.xcp-ng.org/taskinfo?taskID=52176

**Pull request merged**